js/src/files-mfs.js

```diff
@@ -295,7 +298,54 @@ module.exports = (common) => {
             blocks: 2,
             size: 0,
             hash: 'QmVrkkNurBCeJvPRohW5JTvJG4AxGrFg7FnmsZZUS6nJto',
-            cumulativeSize: 216
+            cumulativeSize: 216,
+            withLocality: false,
+            local: undefined,
+            sizeLocal: undefined
+          })
+          done()
+        })
+      })
+
+      it('stat withLocal file', (done) => {
+        if (!withGo) {
+          console.log('Not supported in js-ipfs yet')
+          return done()
```

Comment:
call `this.skip()`. You will have to replace the arrow function by regular function

Reply:
This is the way it's done in all the codebase. Do you prefer to change that here or in a single pass in another PR ?

Reply:
can you change the others first in a PR and update this one to match? thank you :)